RunningOrder/Sprint/Models/Search.swift

```diff
@@ -40,17 +43,32 @@ extension SearchSection: Identifiable {
     var id: String { type.title }
 }
 
-struct SearchItem: Hashable {
-    var name: String
-    var icon: String
-    var type: SearchSection.SectionType
-    var relatedStory: Story?
+enum SearchItem {
+    case story(Story)
+    case epic(String)
+    case filter(String)
+//    case people(String)
```

Comment:
commented code

Reply:
I have another MR ready to change this 😇